### PR TITLE
Interval Timer using Vanilla JS #107

### PR DESCRIPTION
**Pull request by AftabMankapure**
# Pull Request for WebMasterLog 💡

## Issue Title:  Interval Timer using Vanilla JS

- **Name of the project** : Interval Timer
- **Category:** Vanilla JS
- **Name:** Aftab Amin Mankapure
- **GitHub ID:**  https://github.com/AftabMankapure
- **Idenitfy yourself:** JWOC

Closes:  #107 

### Describe the add-ons or changes you've made 📃

Give a clear description of what have you added or modifications made
Added new application interval time 

## Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made

## Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots 📷

i

### CODE REVIEW

**Comment by AftabMankapure** [comment]
@Avdhesh-Varshney  pls check this PR

File: Vanilla-JS-Projects/Intermediate/Interval_Timer/Interval_Timer.webp
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Interval_Timer/Interval_Timer.webp:1]: Replace underscore by hyphen in directory name and also in screenshot name.

File: Vanilla-JS-Projects/Intermediate/Interval_Timer/Interval_Timer.we
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Interval_Timer/Interval_Timer.we:1]: Delete this `.we` extension file. Only screenshot is needed in `.webp` extension.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by Avdhesh-Varshney** [comment]
@AftabMankapure Carefully fill the pr template not just tick all the options if not needed.
And there is not any kind of game in this repo ok.

`Added new game interval time`

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

**Review by Avdhesh-Varshney — APPROVED** [review]
PR Approved 🎉

**Pull request merged**